cves/kernel/CVE-2021-3744.yml

```diff
@@ -19,26 +19,26 @@ curated_instructions: |
   This will enable additional editorial checks on this file to make sure you
   fill everything out properly. If you are a student, we cannot accept your work
   as finished unless curated is properly updated.
-curation_level: 0
+curation_level: 2
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that
   the CVE was created.  Leave blank if no date is given.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2021-09-02
```

Comment:
https://www.openwall.com/lists/oss-security/2021/09/14/1


> CVE-2021-3744: crypto: ccp - fix resource leaks in ccp_run_aes_gcm_cmd()                                                                                                                     
>
> This was reported by Tencent researcher <minihanshen@...cent.com> to
> linux-distros, with disclosure date agreed to September 6th.

was just looking through links from NVD and found this, do you think 2021-09-06 would be more accurate in this case? could be wrong but just double checking!

Reply:
Yeah, good catch